### PR TITLE
MNT Use shared travis config, use sminnee/phpunit

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

**Review by dnsl48 — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -1,39 +1,4 @@
...
-    - php: 5.6
-      env: DB=MYSQL INSTALLER_VERSION=4.2.x-dev PHPCS_TEST=1 PHPUNIT_TEST=1
-    - php: 7.0
-      env: DB=PGSQL INSTALLER_VERSION=4.2.x-dev PHPUNIT_TEST=1
-    - php: 7.1
-      env: DB=MYSQL INSTALLER_VERSION=4.3.x-dev PHPUNIT_COVERAGE_TEST=1
-    - php: 7.2
-      env: DB=MYSQL INSTALLER_VERSION=4.3.x-dev PHPUNIT_TEST=1
-    - php: 7.3
-      env: DB=MYSQL INSTALLER_VERSION=4.4.x-dev PHPUNIT_TEST=1 NPM_TEST=1
```
Comment (dnsl48) [inline .travis.yml:15]: NPM tests are missing

File: .travis.yml [hunk 2]
```diff
@@ -1,39 +1,4 @@
...
-    - php: 7.3
-      env: DB=MYSQL INSTALLER_VERSION=4.5.x-dev PHPUNIT_TEST=1
-    - php: 7.3
-      env: DB=MYSQL INSTALLER_VERSION=4.x-dev PHPUNIT_TEST=1
-
-before_script:
-  - phpenv rehash
-  - phpenv config-rm xdebug.ini
-  - composer validate
-  - composer require silverstripe/installer:"$INSTALLER_VERSION" ezyang/htmlpurifier:* --no-update
```
Comment (dnsl48) [inline .travis.yml:25]: > ezyang/htmlpurifier:*

that package is missing. do we still need it?

**Review by dnsl48 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (emteknetnz) [reply .travis.yml:25]: I've added it in under env global REQUIRE_EXTRA

**Comment by emteknetnz** [comment]
Change package.json `"test": "jest"` to "test": "true" because there are no any javascript tests in the module

**Review by Cheddam — APPROVED** [review]

**Pull request merged**